packages/babel-types/src/definitions/typescript.ts

```diff
@@ -649,23 +649,28 @@ defineType("TSModuleBlock", {
 defineType("TSImportType", {
   aliases: ["TSType"],
   builder: [
-    "argument",
+    process.env.BABEL_8_BREAKING ? "source" : "argument",
     "qualifier",
     process.env.BABEL_8_BREAKING ? "typeArguments" : "typeParameters",
   ],
   visitor: [
-    "argument",
+    process.env.BABEL_8_BREAKING ? "source" : "argument",
     "options",
     "qualifier",
     process.env.BABEL_8_BREAKING ? "typeArguments" : "typeParameters",
   ],
   fields: {
-    argument: process.env.BABEL_8_BREAKING
-      ? validateType("TSLiteralType")
-      : validateType("StringLiteral"),
+    ...(process.env.BABEL_8_BREAKING
+      ? { source: validateType("TSLiteralType") }
+      : { argument: validateType("StringLiteral") }),
```

Comment:
Did some previous change in v8 created `TSLiteralType`? We need revert that change.